### PR TITLE
Crushed

### LINKED ISSUE #74: Crushed proppants
Add functionality for crushed proppants in erosion functions, e.g 

`crushed=false` as input. `if crushed=True: return 3xE`

### PR DESCRIPTION
**Pull request by smolvik1**
Fixes #74

### CODE REVIEW

File: tests/test_erosion.py [hunk 1]
```diff
@@ -171,78 +177,116 @@ def test_alpha_validation(caplog):
...
 # Bend validation 2 to test all versions of gamma
 # Bend validation 3 based on example in DNVGL RP-O501, Aug 2015, chapter 4.7
-bend_validation = [(29.3, 30, 1.5e-5, 1.75, 1, 0.0978, 0.28, 'duplex', pytest.approx(0.255158, abs=10e-6)),
-                   (15, 2, 4e-4, 1.5, 2, .1, .4, 'duplex', pytest.approx(0.003665, abs=10e-6)),
-                   (15, 333.3, 3.4e-4, 1.5, 1, 0.1, 0.25, 'duplex', pytest.approx(0.014322, abs=10e-6))]
-@pytest.mark.parametrize('v_m, rho_m, mu_m, R, GF, D, d_p, material, E', bend_validation)
-def test_bend(v_m, rho_m, mu_m, R, GF, D, d_p, material, E):
-    assert bend(v_m, rho_m, mu_m, R, GF, D, d_p, material=material) == E  # mm/ton
+bend_validation = [
+                    (29.3, 30, 1.5e-5, 1.75, 1, 0.0978, 0.28, 'duplex', False, pytest.approx(0.255158, abs=10e-6)),
```
Comment (knudsvik) [inline tests/test_erosion.py:181]: These validations could possibly been a bit smarter set up (to ease readability).. And possibly to test crushed you could just have used a for loop? Something like 

1. validate results
2. set crushed = True
3. validate all results *3

so you don't have to double up all tests manually?

Comment (knudsvik) [reply tests/test_erosion.py:181]: not that it doesn't work as is, but maybe easier for the future.

File: pysand/erosion.py [hunk 2]
```diff
@@ -40,6 +40,10 @@ def validate_inputs(**kwargs):
                 logger.warning('The model has got negative value(s) of {} and returned nan.'.format(i))
                 return True
 
+    if 'crushed' in kwargs:
+        if not isinstance(kwargs['crushed'], bool):
+            raise exc.FunctionInputFail('{} is not a boolean'.format('crushed'))
```
Comment (knudsvik) [inline pysand/erosion.py:45]: use the f-strings Luke!

Comment (knudsvik) [reply pysand/erosion.py:45]: You could also write a test for this exception?

Comment (smolvik1) [reply pysand/erosion.py:45]: > You could also write a test for this exception?

I thought I did. Line 21-26 in test_erosion.py?

Comment (knudsvik) [reply pysand/erosion.py:45]: Ah yes, good work :)

File: tests/test_erosion.py (hunk 1 above)
Comment (smolvik1) [reply tests/test_erosion.py]: I have now parameterized the crushed input to the test functions. Not 100% on whether it improves readability, but sure does work.

**Pull request merged**